examples/parse_mb2_header_runtime.rs

```diff
@@ -0,0 +1,76 @@
+extern crate multiboot2;
```

Comment:
I'm not sure this is a great example of what users should be doing, and doesn't really showcase this crate. I think we should just stick with the other example.

Reply:
First: The `extern crate multiboot2` was accidentally added by my IDE. Second: I think it would be better to move it to `tests` directory?